Extensions/tagviewer.js

```diff
@@ -242,23 +241,22 @@ XKit.extensions.tagviewer = new Object({
 
 	},
 
-	add_tags: function(by, tags, link, avatar) {
+	add_tags: function(by, tags, link) {
 
 		$("#tagviewer-loading").slideUp('slow', function() { $(this).remove(); });
 
-		var m_html = "<div class=\"tagviewer-tag\">" +
-				"<div class=\"tagviewer-by\">" +
-					"<a target=\"_blank\" href=\"" + link + "\" class=\"tagviewer-by-link\">" + by + "</a>" +
-					"<img class=\"tagviewer-by-avatar\" src=\"" + avatar + "\">" +
-				"</div>" +
-				"<div class=\"tagviewer-tag-tags\">";
+		var m_html = `<div class="tagviewer-tag">` +
```

Comment:
You could cut out the concatenations here since this is a backtick string. It might have some spooky effects on the spacing so I'm also okay with leaving it as-is.

Reply:
i admit i changed these quickly out of rage for seeing so many backslashes, so I probably should've just used single-quotes for most of these lines - i'll change that now

didn't think of spacing though, probably best not to put the ton of whitespace in here...